scanner/scanner.go

```diff
@@ -1,20 +1,49 @@
 package scanner
 
 import (
+	"bytes"
 	"context"
 	"encoding/hex"
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"math"
+	"net/http"
 	"time"
 
+	blockapi "github.com/SkynetLabs/blocker/api"
+	blockdb "github.com/SkynetLabs/blocker/database"
 	"github.com/SkynetLabs/malware-scanner/clamav"
 	"github.com/SkynetLabs/malware-scanner/database"
 	"github.com/SkynetLabs/skynet-accounts/build"
 	"github.com/sirupsen/logrus"
 	"gitlab.com/NebulousLabs/errors"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+const (
+	// malwareTag marks the skylink as blocked by malware-scanner, as opposed to
+	// user-reported malware.
+	malwareTag = "malware-scanner"
 )
 
 var (
+	// BlockerIP is the IP of the blocker service.
+	// Set according to the BLOCKER_IP env var.
+	BlockerIP string
+	// BlockerPort is the port of the blocker service.
+	// Set according to the BLOCKER_PORT env var.
+	BlockerPort string
```

Comment:
f/u

I noticed that we got some of these in the blocker repo as well and we should move this to fields into the `Scanner` as `static` fields. Using exposed global vars like this for configuring things is really annoying in testing because all tests share the same vars and you can't initialize the scanner and configure it without knowing that these are around. So you get race conditions in testing if you set these while running tests in parallel.
It's a lot easier when you just pass them to `New`. If there are only a few configs as arguments and otherwise as a configuration object.

Reply:
F/U https://github.com/SkynetLabs/malware-scanner/issues/10